https://github.com/madler/pigz/issues/111
https://github.com/madler/pigz/commit/907ca0763be4547a9b0cce8c1057217488149744

From 907ca0763be4547a9b0cce8c1057217488149744 Mon Sep 17 00:00:00 2001
From: Mark Adler <madler@alumni.caltech.edu>
Date: Fri, 18 Aug 2023 03:27:12 -0700
Subject: [PATCH] Make pigz compatible with two-component zlib version numbers.

zlib 1.3 (not 1.3.0) broke the zlib_vernum() function in pigz.
This commit fixes that.
--- a/pigz.c
+++ b/pigz.c
@@ -1333,7 +1333,7 @@ local long zlib_vernum(void) {
         }
         ver++;
     } while (left);
-    return left < 2 ? num << (left << 2) : -1;
+    return left < 3 ? num << (left << 2) : -1;
 }
 
 // -- check value combination routines for parallel calculation --

